### PR TITLE
give cached value back

### PR DESCRIPTION
**Pull request by vaslabs**
Hello,
I was following the documentation and realised that this cached value is unused.

Perhaps this is how it was supposed to be? Otherwise the cached value is not needed in the behaviour?

Note: I haven't actually got a local copy for this, sorry if I made a syntax error, I wanted to ask the question first :)

### CODE REVIEW

**Comment by raboof** [comment]
OK TO TEST

File: akka-cluster-typed/src/test/scala/akka/cluster/ddata/typed/scaladsl/ReplicatorSpec.scala [hunk 1]
```diff
@@ -72,8 +72,8 @@ object ReplicatorSpec {
               replicator ! Replicator.Get(Key, Replicator.ReadLocal, getResponseAdapter, Some(replyTo))
               Behaviors.same
 
-            case GetCachedValue(replyTo) ⇒
-              replicator ! Replicator.Get(Key, Replicator.ReadLocal, getResponseAdapter, Some(replyTo))
+            case GetCachedValue(replyTo: ActorRef[Int] @unchecked) ⇒
```
Comment (johanandren) [inline akka-cluster-typed/src/test/scala/akka/cluster/ddata/typed/scaladsl/ReplicatorSpec.scala]: `GetCachedValue` is a concrete type with no type parameter, so `replyTo` can never be anything else than `ActorRef[Int]` so there should be no need for a type in the pattern here.

**Review by johanandren — COMMENTED** [review]
Looks like the right thing to do, but one little issue needs to be fixed

**Review by johanandren — APPROVED** [review]
LGTM

**Review by patriknw — APPROVED** [review]
LGTM, needs a rebase

**Comment by raboof** [comment]
@vaslabs do you want to fix the merge conflict or shall we?

**Comment by vaslabs** [comment]
I can do it, is there a timeframe?

**Comment by johanandren** [comment]
PLS BUILD

**Pull request merged**